cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -48,6 +48,12 @@
 <!-- Malformed links should not show in TagsBlockingFirstPaint artifact -->
 <link rel="stylesheet" href="">
 
+<script>
+  // Set a mobile-friendly viewport only on load.
+  const metaViewport = document.querySelector('meta[name="viewport"]');
+  metaViewport.content = 'width=device-width, initial-scale=1, minimum-scale=1';
```

Comment:
It's really important that we take the viewport `content` from the element (not parsed from the tag in the html) because site can and do change it.

We already do this, but we don't currently have any test coverage for that fact :)

It probably makes sense for the terrible dbw page to have a weird viewport instead of a nice one, but doing so changes test expectations for way too many assertions, so it isn't worth the effort.